src/main/java/it/smartcommunitylab/aac/webauthn/auth/RegisterCredentialAuthenticationToken.java

```diff
@@ -0,0 +1,71 @@
+package it.smartcommunitylab.aac.webauthn.auth;
+
+import java.util.Collection;
+
+import org.springframework.security.authentication.AbstractAuthenticationToken;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.util.Assert;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.webauthn.persistence.WebAuthnUserAccount;
+
+public class RegisterCredentialAuthenticationToken extends AbstractAuthenticationToken {
```

Comment:
what does this authentication token represents? An authentication after login or after registration?